app/__init__.py

```diff
@@ -31,6 +32,13 @@
 from notifications_utils.clients.redis.redis_client import RedisClient
 from notifications_utils.clients.zendesk.zendesk_client import ZendeskClient
 
+socketio = SocketIO(
+    cors_allowed_origins="*",
+    message_queue="redis://localhost:6379",
```

Comment:
This would need to connect to the Redis instance in the environment this runs in, correct?  If so then we'd have to get the service connection string instead and likely figure out how to securely pass the credentials to connect it, too.